code/go/0chain.net/smartcontract/dbs/event/process.go

```diff
@@ -46,12 +46,9 @@ const (
 	TagDeleteSharder
 	TagAddOrOverwriteCurator
 	TagRemoveCurator
-	TagStakePoolReward
-	TagStakePoolBalance
-	TagAddOrOverwriteStakePool
 	TagAddOrOverwriteDelegatePool
-	TagRemoveDelegatePool
-	TagEmptyDelegatePool
+	TagStakePoolReward
```

Comment:
@Sriep changing the order might be a bit risky as all the ids will be reordered and inconsistency in db.
Maybe there is a better way we should avoid it?

Reply:
We have decided to ignore event database inconsistencies for the moment. Hence any 0chain built with an earlier build will break when using this PR. For example, the `sp-info` REST API endpoint is returning stake pool information from the event database, not the MPT so data preceding this PR will be missing. 

Anyway, if we want to make breaking changes, it is best to do it now while it does not matter. The change you hight light involved removing unused tags. 

Regardless of the above the changed enums TagStakePoolReward, TagAddOrOverwriteDelegatePool and TagUpdateDelegatePool had no meaning before this PR, so cannot damage anything by being renumbered.

Still, it's probably good practice to only add Tags when they are needed and to add to the bottom.